kroxylicious-filters/kroxylicious-authorization/src/test/java/io/kroxylicious/filter/authorization/MockFilterContext.java

```diff
@@ -235,7 +235,7 @@ public CloseOrTerminalStage<RequestFilterResult> shortCircuitResponse(@NonNull A
         @Override
         public CloseOrTerminalStage<RequestFilterResult> errorResponse(@NonNull RequestHeaderData header, @NonNull ApiMessage message, @NonNull ApiException apiException)
                 throws IllegalArgumentException {
-            throw new IllegalArgumentException();
+            return new ErrorCloseOrTerminalStage(header, message, apiException, false);
```

Comment:
bit of a pain, we can't easily unit test the `errorResponse` style response since it's a framework responsibility. We'd need the runtime involved. Instead the tests are just checking that we are invoking this with the right sort of error, leaving it up to framework testing to cover `errorResponse` generation.